### PR TITLE
Started working on a test suite.

### PR DESCRIPTION
**Pull request by jbrooksuk**
Not an urgent feature, but it's good to have, especially for testing the API out.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
         "barryvdh/laravel-cors": "^0.5"
     },
     "require-dev": {
+        "graham-campbell/testbench": "~2.0",
```
Comment (GrahamCampbell) [inline composer.json]: 2.1 might be better to start off with

Comment (jbrooksuk) [inline composer.json]: Whoops, I was supposed to check if `2.1` was a `5.1` only version?

Comment (GrahamCampbell) [inline composer.json]: Also, it would make sense to use `^2.1` to match the other notation in here. :)

Comment (jbrooksuk) [inline composer.json]: Updating now.

**Comment by GrahamCampbell** [comment]
> especially for testing the API out.

This will be way better in L5.1. We can actually check error responses are good too.

File: composer.json (hunk 1 above)
Comment (GrahamCampbell) [inline composer.json]: > Whoops, I was supposed to check if 2.1 was a 5.1 only version?

2.1 supports both 5.0 and 5.1. :)

**Comment by jbrooksuk** [comment]
> This will be way better in L5.1. We can actually check error responses are good too.

I'm tempted to upgrade to L5.1 and rebase the repository stuff against it so that I can actually test the API  so that I don't break any output etc in the new version.

**Comment by jbrooksuk** [comment]
> `^2.1`?

Done.

**Comment by jbrooksuk** [comment]
Good to merge? :)

**Pull request merged**